CONTRIBUTING.md

```diff
@@ -1,16 +1,256 @@
+  into smaller ones when possible.
+
+* In the Pull Request form, we recommend that you leave the
+  "Allow edits from maintainers" check box ticked. This will allow maintainer
+  finalizing your PR by pushing in your branch. In general, this speeds up the
+  PR merge in the main repository. Note that this is not an obligation.
+
+* Check if your code follows the [coding conventions][coding-conventions]. For
+  example, you can [uncrustify][uncrustify] a file:
+
+      $ uncrustify -c $RIOTBASE/uncrustify-riot.cfg <your file>
+
+* RIOT provides static test tools to verify the quality of changes (flake8,
+  trailing whitespaces, documentation, etc). These tools are wrapped in a
+  single `make` target: `static-test`.
+  *watch out:* the command below will rebase your branch on your master branch,
+  so make sure they can be rebased (e.g. there's no potential conflict).
+
+      $ make static-test
+
+  Use it before opening a PR to perform last time checks.
+
+* Each commit should target changes of specific parts/modules of RIOT. The
+  commits use the following pattern:
+  `path/to/module/directory: description of changes`.
+  You can use multi-line commit messages if you want to detail more the
+  changes.
+
+* Try to answer reviews as quickly as possible to speed up the review process
+  and avoid stalled PRs.
+
+You can find more information about RIOT development procedure on this
+[wiki page][development-procedures].
+
+[existing-pull-requests]: https://github.com/RIOT-OS/RIOT/pulls
+[coding-conventions]: https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions
+[uncrustify]: http://uncrustify.sourceforge.net
+[development-procedures]: https://github.com/RIOT-OS/RIOT/wiki/Development-procedures
+
+### Git usage
+
+Using git is a bit difficult for newcomers. If you are completely new to git,
+we recommend that you [start by learning it](https://try.github.io/) a bit.
+
+In this section, we give the bare minimum for a better experience with our
+development workflow on GitHub.
+
+[git-scm-getting-started]: https://git-scm.com/book/en/v2/Getting-Started-Git-Basics
+
+#### Setup your local RIOT repository
+
+Before you start modifying code, you need to fork the RIOT upstream repository
+from the [RIOT main github page][riot-github].
+
+If it's your first time with git, configure your name and emails:
+
+    $ git config --global user.name = "<your name here>"
+    $ git config --global user.email = "<your email address here>"
+
+Then clone locally your fork of RIOT (replace `account name` with your actual
+login on GitHub):
+
+    $ git clone git@github.com:<account name>/RIOT.git
+
+You can keep any branch of your local repository up-to-date with the upstream
+master branch with the following commands:
+
+    $ git checkout <branch name>
+    $ git pull --rebase https://github.com/RIOT-OS/RIOT.git
+
+Use it at least before opening a PR. This will ensure the PR can be merged.
```

Comment:
"is mergeable" instead of "can be merged". The first implies that it just isn't possible without rebase (which due to conflicts it might true), the latter implies "it sucks, but we'll manage somehow" to me.